lib/resources/service.rb

```diff
@@ -247,8 +247,12 @@ def info_enabled(status, service_name)
     # $ initctl show-config $job | grep -q "^  start on" && echo enabled || echo disabled
     # Ubuntu 10.04 show-config is not supported
     # @see http://manpages.ubuntu.com/manpages/maverick/man8/initctl.8.html
-    config = inspec.command("#{service_ctl} show-config #{service_name}")
-    enabled = !config.stdout[/^\s*start on/].nil?
+    cmd = inspec.command("#{service_ctl} show-config #{service_name}")
+
+    # if initctl doesn't know `show-config`, try config file as fallback
```

Comment:
I am not 100% sure, but doesn't show-config also returns a non-zero exit code, when the service is not available. Maybe its better to check for the specific version?